### PR TITLE
FAQ.md: update upgrade section

### PR DESCRIPTION
**Pull request by vrutkovs**
Add tips about unavailable upgrades

### CODE REVIEW

**Comment by vrutkovs** [comment]
cc @LorbusChris

File: FAQ.md [hunk 1]
```diff
@@ -104,6 +105,12 @@ Note that nightly builds (from `4.x.0-0.okd`) are pruned every 72 hours.
 If your cluster uses these images, consider [mirroring](https://docs.okd.io/latest/installing/install_config/installing-restricted-networks-preparations.html#installing-restricted-networks-preparations) these files to a local registry.
 Builds from the `stable-4` stream are not removed.
 
+## Why no upgrade edges are available?
```
Comment (elmiko) [inline FAQ.md:108]: minor nit

```suggestion
## Why are no upgrade edges available?
```

**Review by elmiko — COMMENTED** [review]
lgtm, just a minor grammar nit

**Review by LorbusChris — APPROVED** [review]
LGTM

**Review by elmiko — COMMENTED** [review]
i have a feeling TOC link won't work properly, just a heads up

**Comment by vrutkovs** [comment]
Right, fixed it in https://github.com/okd-project/okd/commit/e188aa37866d3f7036ba3a2ce2044bbff1f1c10f

**Pull request merged**